stestr/repository/file.py

```diff
@@ -19,12 +19,22 @@
 import sys
 import tempfile
 
-from six.moves import dbm_gnu as dbm
+import six
 from subunit import TestProtocolClient
 import subunit.v2
 import testtools
 from testtools.compat import _b
 
+if sys.platform != 'win32':
+    from six.moves import dbm_gnu as dbm
+else:
```

Comment:
I like this approach. I was trying to solve for the matrix of [python version x operating system] and not having much luck. But I think doing it this way makes sense and just pick the thing that works best for each OS.